gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -1705,11 +1722,18 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
     real (kind=kind_phys), pointer      :: clouds(:,:,:)      => null()  !<
     real (kind=kind_phys), pointer      :: clw(:,:,:)         => null()  !<
     real (kind=kind_phys), pointer      :: clw_surf(:)        => null()  !<
+    real (kind=kind_phys), pointer      :: clw_surf_land(:) => null()  !<
```

Comment:
There seem to be many variables that are added to both the Sfcprop and Interstitial DDT. Is this necessary? The interstitial variables aren't even allocated as far as I can see.

Reply:
Or, asking the other way round, could we use the Interstitial DDT as much as possible to reduce the memory footprint?

Reply:
Yes, we could. All RUC-specific variables are doubled now: for land and ice. I agree that we should do it the most efficient way.

Reply:
Let's try to use the interstitial type then. We should make this switch a controlled experiment. After finalizing the ccpp-physics PR, run all the tests to create a new baseline and make sure we still get b4b reproducible results for warmstart runs. Then switch the variables only, nothing else, and verify against the baseline from before.